notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<blockquote>The rotation rate of a star at a given point in time tells us about its evolution, its magnetic field, etc.</blockquote><p>This sentence puzzled me - can more be added? It's not immediately clear to me how the rotation rate at one time can tell us about its evolution.</p><p> </p><p>The remainder of the paragraph is great. Small typo: matches a plural (stellar light curves) with singular (offers).</p><p> </p><p>I think the reference to the previous notebook, a description of what it did and where we pick up from there, can be added here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>